test/tests.js

```diff
@@ -0,0 +1,98 @@
+const assert = require('assert');
+const fs = require('fs');
+const _ = require('lodash');
+
+describe('Salesforce Experience module', function () {
+
+  let apos;
+  const testUserId = '0052E00000J72IP';
```

Comment:
this would come from an env var too.

Reply:
Not sure about this one. The test really are for our instance. That's why I can predict the number of groups they're in. I mean I already made the change in a commit, but we shouldn't make this look too much like anyone can replace with their own variables unless we go into specific test data to set up.